pkg/doc/jwt/claims.go

```diff
@@ -0,0 +1,123 @@
+/*
+ *
+ *  * Copyright SecureKey Technologies Inc. All Rights Reserved.
+ *  *
+ *  * SPDX-License-Identifier: Apache-2.0
+ *
+ */
+
+package jwt
+
+import (
+	"encoding/json"
+	"errors"
+	"strconv"
+	"time"
+)
+
+// Claims defines JWT Claims Set (https://tools.ietf.org/html/rfc7519#section-4)
+type Claims struct {
+
+	// Issuer defines iss claim.
+	Issuer string `json:"iss,omitempty"`
+
+	// Subject defines sub claim.
+	Subject string `json:"sub,omitempty"`
+
+	// Audience defines aud claim.
+	Audience Audience `json:"aud,omitempty"`
+
+	// Expiry defines exp claim.
+	Expiry *NumericDate `json:"exp,omitempty"`
+
+	// NotBefore defines nbf claim.
+	NotBefore *NumericDate `json:"nbf,omitempty"`
+
+	// IssuedAt defines iat claim.
+	IssuedAt *NumericDate `json:"iat,omitempty"`
+
+	// ID defines jti claim.
+	ID string `json:"jti,omitempty"`
+}
+
+// NumericDate is a JSON numeric value representing the number of seconds from
+// 1970-01-01T00:00:00Z UTC until the specified UTC date/time, ignoring leap seconds.
+type NumericDate int64
```

Comment:
To me, this is the marshalled type... not the data model type (which I assume should be a Time value).

Reply:
I will check this...